text/0096-rename-tasks-evaluations-hooks.md

```diff
@@ -0,0 +1,47 @@
+# Group Keptn Tasks and Evaluations as Hooks
+
+Proposal to change the way the project refers to Keptn Tasks and Evaluations. Instead, refer to both as Keptn Hooks.
+
+## Motivation
+
+* Make it simpler for a user to understand the core concepts of Keptn Lifecycle Toolkit
+
+## Explanation
+
+Hooks are a fairly standard and well understood software mechanism. So rather than referring to tasks and evaluations - then attempting to explain the difference.
```

Comment:
To me, the distinction between an evaluation and a task is fairly obvious.  What I think is confusing is the apparent inability to do an evaluation then trigger a particular task as a result of the evaluation, although this is a deliberate design decision.  To clarify this, we need a good, complex example of a KeptnApp that includes multiple workloads and uses ArgoCD to implement the "if-this-evaluation-result-then-do-this-logic" -- and I'm not sure whether either the KeptnEvaluation or KeptnTask will be part of that logic.

Reply:
"To me, the distinction between an evaluation and a task is fairly obvious".

In a way, that's exactly my point in this PR. We (as those who know about Keptn) know about these terms. But if you are new to Keptn (and probably new or new-ish) to DORA / SRE / SLIs / SLOs etc. then a `Keptn Task` and a `Keptn Evaluation` will mean very little to you.

The term "hooks" is more generic, more widely used and thus more approachable. We don't remove the `Keptn Task` and `Keptn Evaluation` nomenclature - we just explain that both "Keptn tasks and Keptn evaluations" are types of hooks.